Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -145,6 +149,14 @@ HRESULT InitializeGeometries() {
     }
 };
 
+HRESULT _CGPathGetGeometry(CGPathRef path, ID2D1Geometry** pGeometry) {
+    if (path && pGeometry) {
+        RETURN_IF_FAILED(path->ClosePath());
+        *pGeometry = path->GetPathGeometry().Get();
```

Comment:
You might need to `.CopyTo` this. COM out pointers are expected to be given a new +1.